code/src/App.js

```diff
@@ -1,9 +1,117 @@
-import React from 'react';
+// /////////////// IMPORT //////////////////////// //
+
+import React, { useEffect, useState } from 'react';
+import { ThoughtsList } from 'Components/ThoughtsList/ThoughtsList';
+import { ThoughtsInputBox } from 'Components/ThoughtInputBox/ThoughtsInputBox';
+import { Footer } from 'Components/Footer/Footer';
+
+// /////////////// MAIN APP //////////////////////// //
+
+// The function starts with declaring,
+// the variables and sets the useState value.
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [newMessage, setNewMessage] = useState('');
+
+  // Here we are calling the API and gets the JSON.
+  // The setLoading (true) shows "loading text" if the API call is delayed.
+  // .catch is catching eventual errors and displays them in a consol.log
+  // No options is created for this request its per default a GET
+
+  const fetchThoughts = () => {
+    setLoading(true)
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts')
```

Comment:
Instead of hard-coding the API endpoint https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts in two places, define it as a constant and reuse it. This will make it easier to change the endpoint if needed.